docs/ExperimentConfig.md

```diff
@@ -141,12 +140,17 @@ machineList:
 * __trainingServicePlatform__
   * Description
       
-	  __trainingServicePlatform__ specifies the platform to run the experiment, including {__local__, __remote__}.  
-	* __local__ mode means you run an experiment in your local linux machine.  
+	  __trainingServicePlatform__ specifies the platform to run the experiment, including {__local__, __remote__, __pai__, __kubeflow__}.  
 	
-	* __remote__ mode means you submit trial jobs to remote linux machines. If you set platform as remote, you should complete __machineList__ field.  
+    * __local__ mode means you run an experiment in your local linux machine.  
+	
+	
+    * __remote__ mode means you submit trial jobs to remote linux machines. If you set platform as remote, you should complete __machineList__ field.  
```

Comment:
__remote__ mode: submit trial jobs to remote ubuntu machines, and __machineList__ field should be filed in order to set up SSH connection to remote machine

Reply:
updated to new format